### PR TITLE
[BISERVER-14374] Bug fix; start time field is null for cron jobs (not

### PR DESCRIPTION
**Pull request by peterrinehart**
needed).

### CODE REVIEW

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 46s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **260**, Failures: **0**, Skipped: **10** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pentaho-scheduler-plugin/job/PR-218/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: core/src/main/java/org/pentaho/platform/scheduler2/quartz/QuartzScheduler.java [hunk 1]
```diff
@@ -249,7 +249,7 @@ public static MutableTrigger createQuartzTrigger( IJobTrigger jobTrigger, Quartz
     } else {
       // handle legacy imports
       startDateCal = java.util.Calendar.getInstance();
-      startDateCal.setTime( jobTrigger.getStartTime() );
```
Comment (ddiroma) [inline core/src/main/java/org/pentaho/platform/scheduler2/quartz/QuartzScheduler.java:252]: @peterrinehart There were no logs in the bug, so I am unsure where the NPE occurred. But if it was from "jobTrigger.getStartTime()" where "jobTrigger" is null, wouldn't this still throw an NPE?

Comment (peterrinehart) [reply core/src/main/java/org/pentaho/platform/scheduler2/quartz/QuartzScheduler.java:252]: Good point, I'll check for that too.  That should be "impossible" but there's nothing wrong with being thorough.

**Comment by rmansoor** [comment]
@ddiroma @peterrinehart I have attached the logs in the ticket now

**Comment by buildguy** [comment]
[comment]: <> (FrogbotReviewComment)

<div align='center'>

[👍 Frogbot scanned this pull request and did not find any new security issues.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 8m 3s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **260**, Failures: **0**, Skipped: **10** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pentaho-scheduler-plugin/job/PR-218/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**